kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/Behavior.kt

```diff
@@ -28,7 +28,7 @@ import kotlinx.coroutines.Deferred
  *
  * @see RealRenderContext
  */
-internal data class Behavior<StateT, out OutputT : Any>(
+data class Behavior<StateT, out OutputT : Any> internal constructor(
```

Comment:
Sad to make this public, but don't want to duplicate the code in the test module.